content/references/data-api.md

```diff
@@ -2994,19 +2962,19 @@ A successful response uses the HTTP code **200 OK** and has a JSON body with the
 |:-------------|:-----------------------------------------|:-------------------|
 | `result`     | String                                   | The value `success` indicates that the body represents a successful response. |
 | `count`      | Integer                                  | Number of validators returned. |
```

Comment:
I think "Number of validators returned." should be "Number of validator daily reports returned."?